rs/rosetta-api/icp/tests/system_tests/common/utils.rs

```diff
@@ -7,8 +7,8 @@ use ic_icp_rosetta_client::RosettaClient;
 use ic_ledger_core::block::BlockType;
 use ic_nns_constants::GOVERNANCE_CANISTER_ID;
 use ic_nns_constants::LEDGER_CANISTER_ID;
-use ic_nns_governance::pb::v1::ListNeurons;
-use ic_nns_governance::pb::v1::ListNeuronsResponse;
+use ic_nns_governance_api::pb::v1::ListNeurons;
+use ic_nns_governance_api::pb::v1::ListNeuronsResponse;
```

Comment:
Like I said in my PR description, ic_nns_governance_api::pb::v1::FooRequest is very often a drop in replacement for ic_nns_governance::pb::v1::FooRequest (because the later was just manually copied from the former).